index.html

```diff
@@ -229,10 +229,10 @@ <h3>Skills</h3>
       <section class="contact-form">
         <h2 class="contact-me">Contact me</h2>
         <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need to built or a project that needs coding. I'd love to help with it!</p>
-        <form action="https://formspree.io/ivonnebenites2015@gmail.com" method="post" class="form">
-            <input type="text" name="name" id="name" placeholder="name" class="input">
-            <input type="email" name="email" id="email" placeholder="email" class="input">
-            <textarea name="description" id="" cols="30" rows="10" placeholder="Write your message here.." class="input-text"></textarea>
+        <form action="https://formspree.io/f/mnqwgezj" method="POST" class="form">
+            <input type="text" name="name" id="name" placeholder="name" maxlength="30" class="input" required>
+            <input type="email" name="email" id="email" placeholder="email" class="input" required>
+            <textarea name="description" id="" cols="30" rows="10" placeholder="Write your message here.." maxlength="500" class="input-text"></textarea>
```

Comment:
- [ ] Kindly consider adding the `required` attribute to your `textarea` element, as suggested by the previous reviewer. This attribute ensures that users must fill in the `textarea` field before they can submit the form. By making this field required, you enhance the user experience by prompting them to provide the necessary information.

Reply:
@Strangeal 👋🏾
Very nice to meet you, thank you for the suggestions, and explanations about it.
I appreciate it,  I made the changes you suggested,
Best regards,